Scenes/HUD/GameplayInfo.gd

```diff
@@ -0,0 +1,13 @@
+extends CanvasLayer
+
+var KillLabel = preload("res://Scenes/HUD/share/KillLabel.tscn")
+
+remote func update_killings(killer, victim):
```

Comment:
"update_killings" sounds really weird, I think something like "show_kill" would be more understandable